packages/design-tokens/less/typography.less

```diff
@@ -0,0 +1,31 @@
+/** THIS IS AN AUTOGENERATED FILE **/
+
+@kz-typography-family-mono: "IBM Plex Mono";
+@kz-typography-family-display: "Greycliff CF";
+@kz-typography-family-text: "Inter";
+@kz-typography-weight-bold: 600;
+@kz-typography-weight-regular: 400;
+@kz-typography-display-fontSize-1: 4.5rem;
+@kz-typography-display-fontSize-2: 2.25rem;
+@kz-typography-display-fontSize-3: 1.875rem;
```

Comment:
It slightly irks me that the transform library doesn't convert tokens to different cases. A potential improvement? 🤔